### DISCUSSION TITLE
Chapter 11. 보안과 권한 관리

### DISCUSSION THREAD

**Discussion by crtEvent** [CodeSquad-2023-BE-Study/DB-Study#15] [완료]
Ask a question

**Comment by crtEvent** [comment]
> 암호문을 평문으로 바꾸는 것을 암호화 encryption.평문을 암호문으로 바꾸는 것을 복호화 decryption라고 한다.

반대임

**Reply by new-pow** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by jinny-l** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by leegyeongwhan** [comment]
살짝 다른 주제이긴 하지만 DB의 권한과 보안에 대한 파트이다 보니 , 암호화에 대해 더 학습해 보고싶네요.

**Comment by new-pow** [comment]
단방향 암호화는 비밀번호에 쓰는데 양방향 암호화는 언제 사용하게 될까요?

**Reply by crtEvent** [reply]
전자 서명 같이 재사용성이 있는 정보는 암호화, 복호화 모두 이루어져야 해서 양방향 쓴데요~
